straxen/plugins/bayes_peak_classification.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+import numba
+from scipy.special import logsumexp
+import strax
+import straxen
+
+
+class BayesPeakClassification(strax.Plugin):
+    """
+    Bayes Peak classification
+    Returns the ln probability of a each event belonging to the S1 and S2 class.
+    Uses conditional probabilities and data parameterization learned from wfsim data.
+    More info can be found here xenon:xenonnt:ahiguera:bayespeakclassification
+
+    :param peaks: peaks
+    :param waveforms: peaks waveforms in PE/ns
+    :param quantiles: quantiles in ns, calculate from a cumulative sum over the waveform,
+                      from zero to the total area with normalized cumulative sum to determine the time
+    :returns: the ln probability of a each peak belonging to S1 and S2 class
+    """
+
+    provides = 'peak_classification_bayes'
+    depends_on = ('peaks',)
+    __version__ = '0.0.1'
+    dtype = (strax.time_fields
+             + [('ln_prob_s1', np.float32, 'S1 ln probability')]
+             + [('ln_prob_s2', np.float32, 'S2 ln probability')]
```

Comment:
My only comment is whether we want to write this as the description or if it would be more straightforward for us to use: `ln probability_S1` and `ln probability_S2` or something to illustrate that we mean the log-probability peak is an S1 and S2 respectively.

Reply:
@sophiafarrell if we write `s1_ln_prob`, then when running `event_w_bayes_class` it would add `s1_s1_ln_prob` wich is confusing

Reply:
I just meant the description of the variable! That's all. e.g. proposed line change: 
`+ [('ln_prob_s1', np.float32, 'ln probability S1')]`
so it's minor and not huge if it doesn't get included. Just a suggestion